frontend/src/container/Publications.jsx

```diff
@@ -398,12 +433,25 @@ export class Publications extends React.Component {
   }
 
   handleSaveThreadPublication = async (publication, publicationAsFileList) => {
-    const { props } = this
+    const { props, state } = this
 
     const spaceId = props.currentWorkspace.id
-    const publicationName = this.buildPublicationName(props.user.publicName, props.user.lang)
+    const title = state.newPublicationTitle + ' - ' + this.buildPublicationName(props.user.publicName, props.user.lang)
```

Comment:
Won't this end up with the name being " - {{...}}" when the subject is empty ?

Reply:
yes, should we make it so that it's only "{{...}}"

Reply:
IMHO yes, this would feel weird and we should keep the default title if the subject is empty

Reply:
agreed, also you can use string interpolation

Reply:
done